Honeybadger/screenboard.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Each event is tagged `#honeybadger`, so searching for that tag should bring them up!